db/migrate/20220203135812_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.text :name, null: false
+      t.string :photo
+      t.string :bio
+      t.string :role
+      t.integer :posts_counter, default: 0
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] There is an error in your migrations some have the same name, please fix that
![e](https://user-images.githubusercontent.com/44745855/152488327-92c8836a-3819-4d02-8c5e-3a4f600d33b7.png)